### DISCUSSION TITLE
Send password reset link on admin users table.

### DISCUSSION THREAD

**Discussion by GeminiDev1** [chiiya/filament-access-control#61] [Ideas]
I believe adding a "Send Password Reset Link" option would be very useful for some users in the admin panel. Please see the example below:

![image](https://github.com/user-attachments/assets/0aa3b9f7-1da0-4f87-8cab-ebed895250b6)

Once clicked, an email should be sent to the user, directing them to reset their password.

**Comment by chiiya** [comment]
Hey,

Is the option to request a password reset through the login form not enough?

![image](https://github.com/user-attachments/assets/057f775c-f7a4-4531-a665-b6b8096165e0)

**Reply by GeminiDev1** [reply]
Hey,

From an admin point of view, resetting multiple users' passwords can become tedious when filling out emails. In contrast, clicking "reset password" and sending a reset link is much quicker.
